components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/sync/WorkManagerSyncManager.kt

```diff
@@ -155,6 +159,17 @@ internal class WorkManagerSyncDispatcher(
             notifyObservers { onStarted() }
             isSyncActive = true
         }
+        crashReporter?.recordCrashBreadcrumb(
+            Breadcrumb(
+                category = this::class.java.simpleName,
+                message = "syncing-state-change",
+                data = mapOf(
+                    "active" to isSyncActive.toString(),
```

Comment:
Perhaps we can record active as 1/0, and similarly for engines, with some stable ID mapping e.g. places is 1, logins is 2, etc? That will save some bandwidth on the clients.